engine/Shopware/Kernel.php

```diff
@@ -741,6 +741,7 @@ protected function getKernelParameters()
             'kernel.debug' => $this->debug,
             'kernel.name' => $this->name,
             'kernel.cache_dir' => $this->getCacheDir(),
+            'kernel.cache_dir.product_export' => $this->getCacheDir().'/productexport',
```

Comment:
I guess, the cs-fixer will complain about that :grin: 

```suggestion
            'kernel.cache_dir.product_export' => $this->getCacheDir() . '/productexport/',
```